src/app/components/Results/ResultsList.jsx

```diff
@@ -147,7 +147,6 @@ class ResultsList extends React.Component {
               items={items}
               bibId={bibId}
               getRecord={this.getItemRecord}
-              id="search-result-item-table"
```

Comment:
Because ItemTable still expects it as a prop?

    <table className="nypl-basic-table" id={id}>